src/pages/Book/Book.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import BookedPlace from './BookedPlace';
+import styled from 'styled-components/macro';
+import Empty from './Empty';
+import theme from '../../styles/theme';
+
+const Book = () => {
+  const [bookList, setBookList] = useState(null);
+  const params = useParams();
+  const { userId } = params;
```

Comment:
userId만 사용하는거라면 이렇게 해도 좋을 것 같아요!
```suggestion
  const { userId } = useParams();
```